src/onInit/summarizeData.js

```diff
@@ -1,19 +1,24 @@
 import calculateStatistics from './summarizeData/calculateStatistics';
 import sortRows from './summarizeData/sortRows';
+import clone from '../util/clone';
 import { nest } from 'd3';
 
-export default function summarizeData(key_cols = this.config.key_cols) {
+export default function summarizeData(
+    key_cols = this.config.key_cols,
+    data = this.data.initial_filtered
+) {
     const context = this;
     const fractions = this.config.display_fractions;
     var t0 = this.parent.performance.now();
     //begin performance test
 
+    const data_copy = clone(data); // don't want to change original data object
```

Comment:
running into some object permanence?  always fun to debug

Reply:
haha yes exactly